libs/antsi/src/font.rs

```diff
@@ -100,17 +105,29 @@ pub enum Blinking {
     Fast,
 }
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[non_exhaustive]
+pub enum FontScript {
+    Sub,
+    Super,
+}
+
 #[derive(Debug, Copy, Clone, Eq, PartialEq, Default)]
 pub struct Font {
     weight: Option<FontWeight>,
     family: Option<FontFamily>,
+    // mintty extension
+    script: Option<FontScript>,
 
-    // Value layout: `XXXX_IRHS`
+    // Value layout: `XXÖO_IRHS`
```

Comment:
Should we use the bitflags crate here instead?

Reply:
I considered that but did not want to pull in an additional dependency, although it would be better.

Reply:
I think for now it's fine, but in the case where we hit more than 8 bits, we definitely should pull in a crate, otherwise, we would need to deal with endianness.